nixosConfigurations/ponkila-ephemeral-beta/nix-store-query.txt

```diff
@@ -3697,8 +3583,6 @@
 /nix/store/irx05yvi5lrzya7klgshmgzrp5r7sisz-unittest-check-hook.sh
 /nix/store/aiz2xv0095pwgn4ns9aj20fqbfnakhyi-unit-user-runtime-dir-.service.drv
 /nix/store/9d02bsycdjs4c8a7c8xgz8ijnkc4pld6-unit-user-.service.drv
-/nix/store/aiawrg7jyb6mcg3ayy2y6ds8442xlgss-unit-var-mnt-erigon.mount.drv
-/nix/store/ypq9yjvkd89ijw4pxqjq43jxv6vkcjws-unit-var-mnt-lighthouse.mount.drv
```

Comment:
Is this correct?

Reply:
It shouldn't,  makes no sense

Reply:
Found the cause because tmux and motd is also removed. 

It appears that this does not work https://github.com/ponkila/nixobolus/pull/37/commits/0b8fede211015c680eb2cee2e9eec19456647c3f.

Reverting that with a hotfix.